capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -118,8 +118,22 @@ class ClubPage extends Component {
   render() {
     const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
     const bookTile = this.state.book.authors && <BookSearchTile book={this.state.book} bookLists={this.props.bookLists} updateBookLists={this.props.updateBookLists} />;
-    const owner = this.state.owner && <UserCard removeMember={this.removeMember} club={this.state.club} user={this.state.owner} />;
-    const members = this.state.members.length && this.state.members.map(m => <UserCard key={m.id} user={m} club={this.state.club} removeMember={this.removeMember} />);
+    const owner = this.state.owner &&
+      <UserCard
+        removeMember={this.removeMember}
+        club={this.state.club}
+        user={this.state.owner}
+        updateFriendsList={this.props.updateFriendsList}
+      />;
+    const members = this.state.members.length && this.state.members.map(m =>
+      <UserCard
+        key={m.id}
+        user={m}
+        club={this.state.club}
+        removeMember={this.removeMember}
+        updateFriendsList={this.props.updateFriendsList}
+      />
```

Comment:
These should probably be rendered directly below instead. Don't worry about it in this PR though, just make sure there's an issue documented and add a TODO for now.

Reply:
Sounds good.
Created in #141